test/e2e/nft/import-nft.spec.js

```diff
@@ -0,0 +1,101 @@
+const { strict: assert } = require('assert');
+const { convertToHexValue, withFixtures } = require('../helpers');
+const { SMART_CONTRACTS } = require('../seeder/smart-contracts');
+const FixtureBuilder = require('../fixture-builder');
+
+describe('Import NFT', function () {
+  const smartContract = SMART_CONTRACTS.COLLECTIBLES;
+  const ganacheOptions = {
+    accounts: [
+      {
+        secretKey:
+          '0x7C9529A67102755B7E6102D6D950AC5D5863C98713805CEC576B945B15B71EAC',
+        balance: convertToHexValue(25000000000000000000),
+      },
+    ],
+  };
+  it('should not be able to import an NFT that dose not belong to user', async function () {
+    await withFixtures(
+      {
+        dapp: true,
+        fixtures: new FixtureBuilder()
+          .withPermissionControllerConnectedToTestDapp()
+          .build(),
+        ganacheOptions,
+        smartContract,
+        title: this.test.title,
+      },
+      async ({ driver }) => {
+        await driver.navigate();
+        await driver.fill('#password', 'correct horse battery staple');
+        await driver.press('#password', driver.Key.ENTER);
+
+        // After login, go to NFTs tab, open the import NFT form
+        await driver.clickElement('[data-testid="home__nfts-tab"]');
+        await driver.clickElement({ text: 'Import NFTs', tag: 'a' });
+
+        // Enter an NFT that not belongs to user
+        await driver.fill(
+          '[data-testid="address"]',
+          '0x932Ca55B9Ef0b3094E8Fa82435b3b4c50d713043',
```

Comment:
Wonder if we should use the `contractAddress` like below, but use a `tokenId` that the user does not own

Reply:
Yes good idea! I also switched order of the 2 tests, so the first one test with the valid NFT, and the second one test with the correct address with a wrong token id.